client/my-sites/plans/current-plan/jetpack-checklist/header.js

```diff
@@ -10,7 +10,7 @@ import { localize } from 'i18n-calypso';
 import Card from 'components/card';
 import CardHeading from 'components/card-heading';
 
-const JetpackChecklistHeader = ( { isPaidPlan, translate } ) => (
+const JetpackChecklistHeader = ( { isCompleted, translate } ) => (
```

Comment:
TODO: `isCompleted` prop still needs to be connected to something functionally before this works.